src/components/Training/Modal.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import is from 'styled-is'
+import ReactMarkdown from 'react-markdown'
+import { Padding } from 'styled-components-spacing'
+import StyledLink from '../styledLink'
+import Image from '../Common/Image'
+import remcalc from 'remcalc'
+import { H2, Paragraph } from '../Typography'
+import { Grid, Row, Col } from '../grid'
+import close from '../../images/close.svg'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: #ffffff;
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  z-index: 9999;
+  transition: transform 0.3s, opacity 0.3s;
```

Comment:
just a suggestion, I think it's a good idea for our theme to also expose some values like a default transition duration for example. This way we can reuse that value across the app and change them once if the designers change their minds

Reply:
Addedd:

```
  animations: {
    fast: '200ms',
    normal: '300ms'
  },
```

to the theme and it's now being used everywhere